Library/Formula/mesos.rb

```diff
@@ -12,10 +12,9 @@ class Mesos < Formula
   depends_on :java => "1.7+"
   depends_on :macos => :mountain_lion
   depends_on "maven" => :build
-  # Use our Zookeeper for Yosemite and not the one shipped with Mesos
-  # Remove with next release.
-  # See https://github.com/Homebrew/homebrew/issues/32965
-  depends_on "zookeeper" if MacOS.version == :yosemite
+  depends_on "subversion"
+
+  needs: cxx11
```

Comment:
Apologies, I'm an idiot. I should have said ` needs :cxx11 ` not ` needs: cxx11 `. Doh.